src/test/groovy/edu/ie3/datamodel/utils/ContainerUtilsTest.groovy

```diff
@@ -544,7 +544,7 @@ class ContainerUtilsTest extends Specification {
 		}
 		/* Check impedance of line */
 		graph.getEdge(line.nodeA, line.nodeB).with {
-			assert weight == 0.0016909597866300668d
+			assert weight == 0.0016909597866300665d
```

Comment:
I think the assertion need some overhauling rather than the number 😂

Reply:
![](https://media.giphy.com/media/nroeX6KKmCOnl15FN7/giphy.gif)